src/components/content.jsx

```diff
@@ -1,33 +1,21 @@
 import React from 'react';
 import '../common/main.scss';
 
-const Content = () => {
+const Content = ({ headline, content, iframeLink, iFrameTitle }) => {
     return (
         <React.Fragment>
             <section>
                 <div className='row'>
                     <div className='column'>
-                        <h1 className='content-headline'>Test Headline</h1>
-                        <p>
-                            In the spirit of ideas worth spreading, TED has
-                            created a program called TEDx. TEDx is a program of
-                            local, self-organized events that bring people
-                            together to share a TED-like experience. Our event
-                            is called TEDxJNEC, where x = independently
-                            organized TED event. At our TEDxJNEC event, TEDTalks
-                            video and live speakers will combine to spark deep
-                            discussion and connection in a small group. The TED
-                            Conference provides general guidance for the TEDx
-                            program, but individual TEDx events, including ours,
-                            are self-organized.
-                        </p>
+                        <h1 className='content-headline'>{headline}</h1>
+                        <p>{content}</p>
                     </div>
                     <div className='column'>
                         <div className='content-wrapper'>
                             <iframe
                                 className='tedx-iframe'
-                                title='TEDx'
-                                src='https://www.youtube.com/embed/d0NHOpeczUU'
+                                title={iFrameTitle}
+                                src={iframeLink}
                                 frameborder='0'
                                 allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
                                 allowfullscreen
```

Comment:
Consider this as an optional task:

Can we make this component further reusable by giving it a prop to state if we want the iframe to the left and content to right or vice-versa?

purely an optional task and if you are interested kindly ask for @sohamtamboli 's help